test/itemlistformaction.cpp

```diff
@@ -133,6 +133,40 @@ TEST_CASE(
 	REQUIRE(feed->unread_item_count() == 0);
 }
 
+TEST_CASE(
+	"OP_OPENBROWSER_AND_MARK does not mark read when browser fails",
+	"[ItemListFormAction]")
+{
+	Controller c;
+	newsboat::View v(&c);
+	TestHelpers::TempFile browserfile;
+
+	std::string test_url = "http://test_url";
```

Comment:
It'd be nice to mark this `const` to indicate that the test doesn't change this variable.